person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  def initialize(_id, age, name = 'unknown', parent_permission: true)
```

Comment:
- [ ] it's not necessary to include "id" as a parameter. The id is part of the attributes and it's generated automatically.